dwnl_from_wiki.py

```diff
@@ -0,0 +1,20 @@
+"""
+Модуль реализует взаимодействие с API
+Wikipedia
+"""
+import wikipedia as wiki
+
+
+def get_wiki_info(word: str):
+    wiki.set_lang('ru')
+    print(word)
+    """
+    :param word:искомое слово для поиска
+    :return:
+    результат поиска по ключевому слову
+    """
+    info = f'По запросу {word} ничего не найдено'
```

Comment:
```suggestion
    info = f'По запросу {word} ничего не найдено'
```
Стоит переименовать **info** в **query_result**, т.к. **info** не очень четко описывает, что конкретно возвращается

Reply:
Переименовано